src/components/Sections/SectionsProgress.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import { Button, Grid, Icon, List, Progress } from 'semantic-ui-react'
+import { Progress as ProgressType, SectionAndPage, SectionsStructureNEW } from '../../utils/types'
+import strings from '../../utils/constants'
+
+interface SectionsProgressProps {
+  sections: SectionsStructureNEW
+  resumeApplication: (location: SectionAndPage) => void
+}
+
+const SectionsProgress: React.FC<SectionsProgressProps> = ({ sections, resumeApplication }) => {
+  const getIndicator = ({ completed, valid }: ProgressType) => {
+    return completed ? (
+      <Icon name={valid ? 'check circle' : 'exclamation circle'} color={valid ? 'green' : 'red'} />
+    ) : (
+      <Icon name="circle outline" />
+    )
+  }
+
+  const getProgress = ({ doneRequired, doneNonRequired, totalSum, valid }: ProgressType) => {
+    const totalDone = doneRequired + doneNonRequired
+    return totalDone > 0 && totalSum > 0 ? (
+      <Progress
+        style={{ width: 200 }}
+        percent={(100 * totalDone) / totalSum}
+        size="tiny"
+        success={valid}
+        error={!valid}
+      />
+    ) : null
+  }
+
+  // TODO: Use correct firstIncomplete sections
+  const firstIncomplete: SectionAndPage = { sectionCode: 'S1', pageName: 'Page1' }
```

Comment:
You should be able to add this now from using my PR#376

Reply:
Great. I'll rather wait for it to be merged into master first - to avoid merging confusion between branches

Reply:
Done: a6d10ce